NSS-Enthusiastic-Camels/WorkforceManagement
# first commit

Link to Ticket
closes # Link

Description of Proposed Changes
Commented code for Department Comtroller
Steps to Test
git fetch --all
git checkout gw-createdepartment 
Run BangazonWorkforce.
-Browser opens.
-Press Departments tab in navbar.

-All departments are listed
-User can create a new Department


-In visual studio press the test tab.
-Move mouse down to the Windows tab.
-Press on Test Explorer.

-in text explorer, right click on 'Get_IndexReturnsSuccessAndCorrectContentType'
-press on 'run selected test'.

-in text explorer, right click on 'Post_CreateAddsDepartment()'
-press on 'run selected test'.

-see that a green check mark appears next to the test.

Impacted Areas in Application
List general components of the application that this PR will affect:

DepartmentController.cs
Mentions @username
@stjepad 
@mateusvanhalen
@ThraxHamilton

Definition of Done
The project must be fully documented. This includes the following:
Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
Every class must be documented with purpose, author, and methods.
Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
The project must be able to run fully, and without errors, on each teammate's system.
Fulfills every requirement.
Every line of code has been peer reviewed.
For projects that require unit testing, core functionality must be identified and have at least one test for each.
